src/main/java/de/zalando/aruha/nakadi/domain/Cursor.java

```diff
@@ -8,6 +8,7 @@
 public class Cursor {
 
     public static final String BEFORE_OLDEST_OFFSET = "BEGIN";
+    public static final String AFTER_NEWEST_OFFSET = "END";
```

Comment:
Agree with NEWEST, and again, I think that we need to decide which values to use: https://github.com/zalando/nakadi/pull/270#discussion-diff-66392347R20